### DISCUSSION TITLE
Apple Music Radio Stations

### DISCUSSION THREAD

**Discussion by JDogg016** [music-assistant/support#2947] [Q&A] [answered]
First off, the Music Assistant is fantastic!  I find it so easy to create automation using search and playing to groups of speakers. Thank you so much!

I cannot find out how to access 'Apple Radio' stations, specifically the few radio stations offered exclusively by Apple, such as Apple Music Hits, Apple Music Country, and Apple Music One. Does anyone have any suggestions about how to do this?

**Comment by OzGav** [comment]
@MarvinSchenkel ?

**Comment by chasut** [comment]
I don't think there is a way currently.  At least as far as I have found.  But, there is a pretty good workaround available that I have been using even before moving to MA.  Apple also manages playlists that mirror those stations.  They update the tracks regularly, so put it on shuffle and you have a good radio station simulator.  On the plus side, there is a lot more variety in selection than those that have an apple radio station.
Certainly would love for the stations to be supported as well, as the workaround does not cover the "hosted" variety of stations.

**Comment by MarvinSchenkel** [comment, marked as answer]
There's currently no way to sync radio stations from providers to MA. The playlist workaround should work OK. We have talked about adding radios to MA but this currently does not have a high priority

**Reply by gldrenthe89** [reply]
Any update on the priority of this? As BluOS is missing propper Apple Music support i wanted to use MA for this but found out the Apple Music Radio streams are missing.

**Reply by MarvinSchenkel** [reply]
@marcelveldt I don't think we can already use external "dynamic playlists" can we?

**Reply by OzGav** [reply]
I believe Spotify's weekly playlists work if you don't use a personal clientID

**Reply by gldrenthe89** [reply]
> I believe Spotify's weekly playlists work if you don't use a personal clientID

I dont use Spotify. I was asking this for pure Apple Music One radio stations as they expanded to 6 genres, and i use them a lot 😉

**Reply by OzGav** [reply]
Yes but I was responding to Marvin

**Comment by handswork** [comment]
I would love to have Apple Music Radio as well

**Comment by Jokimbe** [comment]
I understand it's not a high priority when there's probably not a lot of us that would use it :)
But would you mind sharing if you've already tried? If so, did it require too many changes to existing code?
I wouldn't mind taking a look at it, getting the stations seems easy enough, but simply passing the station id to the player the same way as a playlist or song id doesn't work. But you probably already know that :P, so I was wondering how far you'd gotten with it, if you tried?
